libexec/rc/rc.d/ldconfig

```diff
@@ -43,7 +43,14 @@ ldconfig_start()
 		machine_arch=$(sysctl -n hw.machine_arch)
 
 		case ${machine_arch} in
-		aarch64c|riscv64c)
+		aarch64c)
+			ldconfigc18n_hints_file="/var/run/ld-elf-c18n.so.hints"
+			cp /var/run/ld-elf.so.hints ${ldconfigc18n_hints_file}
+			_LDC="/usr/lib/c18n ${_LDC}"
+			startmsg 'Library-based compartmentalization ldconfig path:' ${_LDC}
+			${ldconfig} ${_ins} -f ${ldconfigc18n_hints_file} ${_LDC}
+			;& # Fallthrough
```

Comment:
The use of shell fallthrough is extremely uncommon (I found a single case outside the sh source tree), but fine with a comment.  The one other cause uses `FALLTHROUGH` like in C code rather than `Fallthrough`.